uniro_backend/src/main/resources/application-dev.yml

```
@@ -41,4 +38,7 @@ management:
         enabled: true
 
 cors:
-  allowed-origins: ${allowed-origins}
\ No newline at end of file
+  allowed-origins: ${allowed-origins}
+
+jwt:
+  secret: ${JWT_SECRET}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)